vic/drivers/shared_all/src/print_library_shared.c

```diff
@@ -438,18 +446,18 @@ print_layer_data(layer_data_struct *ldata,
     size_t i;
 
     fprintf(LOG_DEST, "layer_data:\n");
-    fprintf(LOG_DEST, "\tCs   : %.4f\n", ldata->Cs);
-    fprintf(LOG_DEST, "\tT    : %.4f\n", ldata->T);
-    fprintf(LOG_DEST, "\tevap : %.4f\n", ldata->evap);
+    fprintf(LOG_DEST, "\tCs   : %f\n", ldata->Cs);
+    fprintf(LOG_DEST, "\tT    : %f\n", ldata->T);
+    fprintf(LOG_DEST, "\tevap (flux): %f\n", ldata->evap);
```

Comment:
@tbohn where is the duplicated line? (I'm reordering this section so that `layer.evap` and `layer.bare_evap_frac` will be separated from the other state variables)